cvat-sdk/cvat_sdk/core/downloading.py

```diff
@@ -58,6 +61,17 @@ def download_file(
             except ValueError:
                 file_size = None
 
+            if output_path.is_dir():
+                content_disposition = next(
+                    part.strip()
+                    for part in response.headers.get("Content-Disposition", "").split(";")
+                    if part.strip().startswith("filename=")
+                )
+
+                filename = Path(content_disposition.split("=", maxsplit=1)[1].strip('"'))
+                filename = filename.with_stem(normalize_filename(filename.stem))
```

Comment:
I don't agree with the `normalize_filename` approach. This function promises to save the file with the "server-generated name". So we should do that, rather than mangling the name in arbitrary ways. If the generated name is not a valid filename, then we should throw an exception. Presumably this will never happen with a legitimate server.

Reply:
Ok, this is done on the server anyway.